airflow/providers/slack/operators/slack.py

```diff
@@ -161,16 +161,26 @@ class SlackAPIFileOperator(SlackAPIOperator):
 
     .. code-block:: python
 
-        slack = SlackAPIFileOperator(
-            task_id="slack_file_upload",
-            dag=dag,
-            slack_conn_id="slack",
-            channel="#general",
-            initial_comment="Hello World!",
-            filename="hello_world.csv",
-            filetype="csv",
-            content="hello,world,csv,file",
-        )
+       # Send file with filename and filetype
+       slack = SlackAPIFileOperator(
+          task_id="slack_file_upload",
+          dag=dag,
+          slack_conn_id="slack",
+          channel="#general",
+          initial_comment="Hello World!",
+          filename="hello_world.csv",
+          filetype="csv"
+       )
+
+       # Send file content
+       slack = SlackAPIFileOperator(
+          task_id="slack_file_upload",
+          dag=dag,
+          slack_conn_id="slack",
+          channel="#general",
+          initial_comment="Hello World!",
+          content="file content in txt",
+       )
```

Comment:
This is also indented wrong. (Sorry about being picky; people are known to copy-paste the examples and this would be very annoying.)

```suggestion
        # Send file with filename and filetype
        slack = SlackAPIFileOperator(
            task_id="slack_file_upload",
            dag=dag,
            slack_conn_id="slack",
            channel="#general",
            initial_comment="Hello World!",
            filename="hello_world.csv",
            filetype="csv"
        )

        # Send file content
        slack = SlackAPIFileOperator(
            task_id="slack_file_upload",
            dag=dag,
            slack_conn_id="slack",
            channel="#general",
            initial_comment="Hello World!",
            content="file content in txt",
        )
```